doc/deploying_gcs_on_bare_metal_machines.md

```diff
@@ -0,0 +1,196 @@
+### Deploying kubernetes + GCS on bare metal machines
+
+#### Cluster:
+
+Total number of nodes: 3 {Node1, Node2, Node3}
+
+#### Node configurations:
+
+OS: Fedora 29<br/>
+RAM: 128 GB<br/>
+CPUs: 24<br/>
+
+### Deploying Kubernetes on bare metal machines:
+
+##### Cluster details:
+
+Total number of nodes : 3 {Node1: kube1, Node2: kube2, Node3: kube3}<br/>
+Master nodes : 1 {kube1}<br/>
+Kube nodes: 3<br/>
```

Comment:
can you mention that the master node is again reused here? Even though its mentioned below, i think its better to clear the confusion.